### DISCUSSION TITLE
Mix curve using lower temp when idle

### DISCUSSION THREAD

**Discussion by darkslid3** [Rem0o/FanControl.Releases#2508] [Q&A] [answered]
I installed and set up the app using Jayz2C's guide, I created a CPU, GPU and Mix curve.

My case fans (3X AIO top exhaust, 3X front intake, 1X rear exhaust) all run through an ARGB fan hub, AIO header connected to CPU_OPT running at full speed.

I set case fans to use the Mix curve so they would ramp up according to higher load of the GPU/CPU. When benching both I do see that the app does select the correct curve, i.e. if I run Cinebench CPU Temp curve is applied, running Heaven then GPU curve is applied.

However, for some reason when the system is idle or just being used casually, the app is using the curve of the lower temp of the two, and not the higher, which is weird because it's set to Max:

![image](https://github.com/Rem0o/FanControl.Releases/assets/169685897/42b465af-80e0-492c-af5c-fafb1479f152)

As you can see here CPU temp is 38C while GPU is 27C and the app decides to run the "GPU Temp" curve in Mix mode. Using the latest 189 .NET 8 version of the app.

Anyone has an idea why? Thanks in advance.

**Comment by ionGL** [comment]
The max mix curve looks at the highest %, not the temp. In your case, the "GPU Temp" curve has the higher %, so obviously it will set its %.

You want to use the gpu temp curve just for the gpu and create separate curves with temp sources gpu and cpu for the case fans. Adjust and mix them as needed.

**Comment by darkslid3** [comment]
Thanks, that makes sense, for some reason in Jay's vid I thought he said that curve will be decided by Temp and not by curve %. 

> You want to use the gpu temp curve just for the gpu and create separate curves with temp sources gpu and cpu for the case fans. Adjust and mix them as needed.

I think that's exactly what I did? Have a look at the screenshot, I have 2 curves for CPU and GPU, GPU is using GPU and Case fans using the mixed CPU/GPU curves.

**Reply by ionGL** [reply, marked as answer]
Graph curve follows temp, the max mix sets the highest % from the curves it manages.
As that gpu temp curve has the highest %, the case fans will follow that. 47% could be unnecessary at idle for them.
It's not optimal to use the same graph curve for the gpu and case fans.
Just add another graph curve for the case fans, responding to the gpu temp. You mix it with the cpu temp graph.
You will get it once you play around and figure the program out.
